integration-tests/smoke/ccip/ccip_test.go

```diff
@@ -18,8 +17,7 @@ import (
 
 func TestInitialDeployOnLocal(t *testing.T) {
 	t.Parallel()
-	lggr := logger.TestLogger(t)
-	tenv, _, _ := testsetups.NewLocalDevEnvironmentWithDefaultPrice(t, lggr, nil)
+	tenv := changeset.NewMemoryEnvironmentWithJobsAndContracts(t, logger.TestLogger(t), 2, 4, nil)
```

Comment:
not the best test to convert, may be try with fee boosting test. that has the longest running time. This one is just a basic test and should stay in docker. We already have a similar test in memory

Reply:
Also I see the runtime is not improved much( it's > 9min) in CI any idea why?